images/linux/scripts/installers/Validate-Toolset.ps1

```diff
@@ -11,11 +11,15 @@ function Run-ExecutableTests {
     )
 
     foreach ($executable in $Executables) {
+        $versionCommand = "--version"
+        If ($executable -eq "bin/go") {
```

Comment:
What  if we add more tools in future and they can't be tested with `--version`? I don't think that we should write if for every tool. Is there any more generic solution?

Reply:
Can we put this information to 
```
$toolsExecutables = @{
    Python = @("python", "bin/pip")
    node = @("bin/node", "bin/npm")
    PyPy = @("bin/python", "bin/pip")
    go = @("bin/go")
}
```
somehow?

Reply:
Done.